documentation/Setup-TLS-on-Docker-Container.md

```diff
@@ -0,0 +1,50 @@
+## TLS connection for REST-adapter-service
+
+When the REST Adapter Service communicates as an HTTP client over HTTPS, the server’s TLS certificate must be trusted. In the consumer gateway role, this means importing the Security Server’s internal TLS certificate into the REST Adapter Service’s Java trust store. In the provider gateway role, it means importing the REST service’s TLS certificate.
+
+### Adapter acting as a client for TLS
+
+You need to get the server certificate (e.g. from an Security Server administrator) and import it into a java truststore. You can use this command:
+
+```shell    
+keytool -importcert -noprompt -alias <meaningful-alias> -file <certificate-name>.cer -keystore truststore.p12 -storetype PKCS12 -storepass '<password>'
+```
+
+Then you need to mount the truststore into the Docker container and configure Java to use that truststore along with the correct password.
+To do so, please add this section to your Docker run command:
+
+```shell
+-v <path to truststore on host>:<path to truststore on container>:ro \
+-e JAVA_TOOL_OPTIONS="-Djavax.net.ssl.trustStore=<path to truststore on container> -Djavax.net.ssl.trustStorePassword=<password>"
+```
+
+### Adapter acting as a client for mTLS 
```

Comment:
I believe the Security Server certificate still needs to be trusted as well. I think it would be good to mention that the previous step is also still required and the final keystore should then include both the Security Server certificate from the previous step as well as the client certificate in this step.

We could even consider dropping this to a heading 4 so that it is logically nested under the previous step.